src/basic/virt.c

```diff
@@ -30,6 +30,7 @@ static const char *const vm_table[_VIRTUALIZATION_MAX] = {
         [VIRTUALIZATION_VMWARE]    = "VMwareVMware",
         /* https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/reference/tlfs */
         [VIRTUALIZATION_MICROSOFT] = "Microsoft Hv",
+        [VIRTUALIZATION_MICROSOFT] = "Micr",
```

Comment:
Why? Please explain this change

Reply:
On Microsoft's virtual machine(Hyper-v).  The currently recognized virtual machine type is "vm-other".
But sig.text value in the "detect_vm_cpuid(void)" function is "Micr".   
So "Micr" is the abbreviation of "Microsoft".

Reply:
I cannot reproduce this, it works fine here, `sig.text` contains `Microsoft Hv` as expected:

```
$ systemd-detect-virt --version
systemd 247 (247.3-6~bpo10+1)
+PAM +AUDIT +SELINUX +IMA +APPARMOR +SMACK +SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +ZSTD +SECCOMP +BLKID +ELFUTILS +KMOD +IDN2 -IDN +PCRE2 default-hierarchy=hybrid
$ SYSTEMD_LOG_LEVEL=debug systemd-detect-virt
Failed to read $container of PID 1, ignoring: Permission denied
Found container virtualization none.
No virtualization found in DMI
UML virtualization not found in /proc/cpuinfo.
Virtualization found, CPUID=Microsoft Hv
Found VM virtualization microsoft
microsoft
```

Reply:
You are right, it was my mistake that caused the wrong type of virtual machine to be obtained.